frontend/shared/browser-tests/utils/settings.ts

```diff
@@ -1,4 +1,7 @@
 export const getUrl = (baseUrl = '', path?: string): string =>
   `${baseUrl}${path?.startsWith('/') ? path : path ? `/${path}` : ''}`;
 
-export const SuomiFiAuthorizationUrl = 'https://testi.apro.tunnistus.fi/';
+export const SuomiFiAuthorizationUrls = [
```

Comment:
Interestingly enough, it seems that sometimes suomifi test environment points to a different address, so here is the list all possible addresses witnessed so far. These are used by browser tests.